Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -292,6 +293,8 @@
     <Content Include="SamplePages\BluetoothLEHelper\BluetoothLEHelper.png" />
     <Content Include="SamplePages\BackdropBlurBrush\BackdropBlurBrush.png" />
     <Content Include="SamplePages\Blur\BlurBehavior.png" />
+    <Content Include="SamplePages\CameraPreview\Camera.png" />
```

Comment:
Can you replace these two with these (CameraPreview and CameraHelper respectively) :

![cameracontrol](https://user-images.githubusercontent.com/711864/39099178-b4598f80-462a-11e8-904c-56a9687cc870.png)

![camerahelper](https://user-images.githubusercontent.com/711864/39099179-b652bf8c-462a-11e8-9317-61c546c9fb4c.png)

Reply:
Changed icons